src/openvic-simulation/modifier/ModifierManager.hpp

```diff
@@ -1,15 +1,12 @@
 #pragma once
 
+#include <string_view>
 #include "openvic-simulation/modifier/Modifier.hpp"
```

Comment:
```suggestion
#include <string_view>

#include "openvic-simulation/modifier/Modifier.hpp"
```
Newline between include groups